play-json/jvm/src/test/scala/play/api/libs/json/ReadsSpec.scala

```diff
@@ -756,28 +756,28 @@ class ReadsSpec extends org.specs2.mutable.Specification {
 
     "parse long string" in {
       val input = JsString(longNumberString)
-      reads(input).must(beLike {
+      reads(input) must beLike {
         case JsSuccess(_, _) => ok
-      })
+      }
     }
 
     "not parse string exceeding length limit" in {
       val input = JsString(longNumberString + "1")
-      reads(input).must(beLike {
+      reads(input) must beLike {
         case JsError((_, JsonValidationError("error.expected.numberdigitlimit" :: Nil) :: Nil) :: Nil) => ok
-      })
+      }
     }
 
     "parse string with acceptable scale" in {
       val numberString = s"1E+${settings.scaleLimit}"
-      reads(JsString(numberString)).must_===(JsSuccess(BigDecimal(numberString)))
+      reads(JsString(numberString)) mustEqual JsSuccess(BigDecimal(numberString))
```

Comment:
`mustEqual` is not `must_===`

```suggestion
      reads(JsString(numberString)) must_=== JsSuccess(BigDecimal(numberString))
```

Reply:
Done.
So now all other asserts use different style (not infix). So if you insist that infix style is preferred then other places must be formatted in similar way. That will be a change spanning multiple files which are not related to this particular fix. That is what I meant saying we'd better change formatting in separate MR. I could volunteer for that after when this MR is merged.

Reply:
That's ok, Thx